.github/workflows/npm-codescan-complete.yml

```diff
@@ -67,7 +72,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: ['lts/*']
+        node-version: ${{ fromJSON(inputs.node_versions) }}
```

Comment:
awesome! I was trying to figure out a good way to do this but you nailed it! @ak-slalom this may be helpful to you since you.